code/modules/client/client procs.dm

```diff
@@ -176,7 +176,10 @@ var/next_external_rsc = 0
 
 	while (query.NextRow())
 		player_age = text2num(query.item[2])
-		break
+		return
+	//player not found.
+	player_age = 0

```

Comment:
Player age is days. It would be far too costly in terms of cpu to be doing a search thru the connection logs to count how many connections they had. 

I'm just putting a log entry for when it can't find them in the database.

Now, a log entry for them connecting, every connection, for x days seems like it could get too spammy the moment you have mutiple new people.

Every round, 2 or 3 people triggering a admin message? and what happens the next time some medium sized website talks about ss13? 15 people triggering an admin message every round?